PhpAmqpLib/Wire/IO/StreamIO.php

```diff
@@ -256,7 +256,7 @@ public function write($data)
             }
 
             set_error_handler(array($this, 'error_handler'));
-            $buffer = fwrite($this->sock, $data);
+            $buffer = fwrite($this->sock, $data, 8192);
```

Comment:
What's the significance of 8192 being the number chosen?  Can we make this a constant or something to help describe what this number is?  Just a thought.